package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "oppia",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
```

Comment:
Omit this and other empty fields if not needed

Reply:
Done.